crates/markdown_preview/Cargo.toml

```diff
@@ -19,19 +19,21 @@ anyhow.workspace = true
 async-recursion.workspace = true
 collections.workspace = true
 editor.workspace = true
+fs.workspace = true
```

Comment:
Very strange that `machete` is not reacting, but new code dose not use this, it seems?

Let's remove it if we can, otherwise curious to get some context.

Reply:
Ahh, `Cargo.lock` seems to be not updated even, maybe that's why the machete is fine.

Reply:
Indeed, sorry.